Userland/Libraries/LibVideo/DecoderError.h

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2022, Gregory Bertilson <zaggy1024@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Error.h>
+#include <AK/Format.h>
+#include <AK/String.h>
+#include <errno.h>
+
+namespace Video {
+
+struct DecoderError;
+
+template<typename T>
+using DecoderErrorOr = ErrorOr<T, DecoderError>;
+
+enum class DecoderErrorCategory : u32 {
+    Unknown = 0,
```

Comment:
Enums start with 0 by default, so there is no need for the `= 0`.

Reply:
Fixed.